build.sbt

```diff
@@ -2,7 +2,7 @@ lazy val root = (project in file(".")).enablePlugins(BuildInfoPlugin).settings(
 
   name := "jacoco4sbt",
   organization := "de.johoop",
-  version := "2.2.0",
+  version := "2.3.0",
```

Comment:
Done.

I've updated all the tests to pull in the current build so we don't mistakingly run tests against a previous version. I left the `addPlugin` statement in the readment on `2.2.0` since it's the current released version.